src/main/java/org/terasology/navgraph/BitMap.java

```diff
@@ -114,6 +141,11 @@ public String toString() {
         return sb.toString();
     }
 
+    /**
+     *
```

Comment:
There's no comment here, but it could use one (or at least get rid of the extra line)